website/src/data/users.js

```diff
@@ -17,12 +17,6 @@ const users = [
     infoLink: 'https://beamery.com/',
     pinned: true,
   },
-  {
-    caption: 'Jump',
```

Comment:
rip

Reply:
technically you could replace it with `Nuvi` but we'd probably have to get their consent, and that's not something I can do at this moment :D

Reply:
Let's ask the CEO via a Glassdoor DM? 😄